### DISCUSSION TITLE
New Install - iPhone 16 Live Photos Issue

### DISCUSSION THREAD

**Discussion by satguys** [immich-app/immich#14492] [Q&A]
I just installed Immich earlier and the week on Ubuntu using the Snap.  I am running the version v1.121.0.

I successfully imported all the photos from my iPhone 16 Pro Max, however all the Live Photos that I have taken with my iPhone 16 show as Error Loading Image.  If I point to the Play Icon in the top right of the error loading image graphic, I can see my moving live image.

This is only happening on Live Images I have taken with my new iPhone 16, the live images I have taken with my old iPhone 13 Pro Max show thumbnails fine. 

I have tried rebuilding thumbnails with no success.  Since its just happening on new images from iPhone 16 Pro Max, I am guessing the issue is a setting on my iPhone and not an issue with Immich.

Has anyone had this issue like this?  How do I fix it?

Thanks in advance!

**Comment by bo0tzz** [comment]
You're using an unsupported installation method, and this sort of issue is often based in mismatched system dependencies and such, so this isn't something we can really help with.

**Comment by alextran1502** [comment]
I suggest you use the official docker-compose method to install Immich
